client/components/payments-activity/survey/style.scss

```
@@ -0,0 +1,59 @@
+.wcpay-payments-activity__survey {
+	$inner-container-max-width: 513px;
+
+	position: relative;
+	width: 100%;
+
+	.components-button.has-icon {
+		min-width: 32px;
+		height: 32px;
+	}
+
+	.survey_container {
+		margin: 0 auto;
+		max-width: $inner-container-max-width;
+		display: flex;
+		gap: 10px;
+		justify-content: center;
+		align-items: center;
+
+		span[role='img'] {
+			line-height: 32px;
+			font-size: 20px;
+		}
+
+		&__emoticons {
+			display: flex;
+			justify-content: flex-end;
+
+			.selected {
+				background-color: $wp-blue-0;
+			}
+		}
+
+		@media ( max-width: 600px ) {
```

Non-blocking feedback - 

I think we have a [breakpoint](https://github.com/Automattic/woocommerce-payments/blob/3666e0d1109f35386a45e88323f45cb45c3dc513/client/stylesheets/abstracts/_breakpoints.scss#L8) of 660px. 

We may also consider using our built-in mixin for responsive.

```suggestion
		@include breakpoint('<660px') {
		...
```

---

👏 Thanks, resolved in c20d8dedf2e7b7e436d77cc683b2025b526b5f76

---

c20d8dedf2e7b7e436d77cc683b2025b526b5f76